arc/cli/commands/run.py

```
@@ -784,101 +787,83 @@ async def _execute_with_orchestrator(
+        # Rich progress display with batch processing metrics
+        with Progress(
+            SpinnerColumn(),
+            TextColumn("[progress.description]{task.description}"),
+            BarColumn(),
+            TextColumn("[progress.percentage]{task.percentage:>3.0f}%"),
+            TimeRemainingColumn(),
+            TextColumn("[dim]({task.fields[status]})[/dim]"),
+            console=console,
+        ) as progress:
+            task = progress.add_task(
+                "Executing scenarios...", 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)